### ISSUE TITLE
Where's the code of this project?

### ISSUE THREAD

**Issue by Korayem** [chaskiq/chaskiq-react-native#1]
It's empty!?

**Comment by michelson** [comment]
yes, we indeed start the repo because someone shows some interest to do it but then nothing happened,
maybe you could help us :)

**Comment by premiumwatchdevice** [comment]
> yes, we indeed start the repo because someone shows some interest to do it but then nothing happened, maybe you could help us :)

Can you update this project?
